src/persist/src/indexed/columnar/parquet.rs

```diff
@@ -129,12 +115,12 @@ pub fn decode_trace_parquet<R: Read + Seek>(r: &mut R) -> Result<BlobTraceBatch,
     Ok(ret)
 }
 
-fn encode_parquet_kvtd<W: Write>(
+fn encode_parquet_kvtd<'a, W: Write, I: IntoIterator<Item = &'a ColumnarRecords>>(
```

Comment:
we don't need this `I: IntoIterator` anymore right? I think both callers just have a `&[ColumnarRecords]`